app/uk/gov/hmrc/tariffclassificationfrontend/controllers/PdfDownloadController.scala

```diff
@@ -47,6 +47,14 @@ class PdfDownloadController @Inject()(authenticatedAction: AuthenticatedAction,
     }
   }
 
+  def liabilityPdf(reference: String): Action[AnyContent] = authenticatedAction.async { implicit request =>
+    caseService.getOne(reference) flatMap {
+      case Some(c: Case) if c.decision.isDefined => generatePdf(decision_template(c, c.decision.get), s"LiabilityDecision_$reference.pdf")
```

Comment:
worth checking that c.application.isLiability?

Reply:
The original endpoint for rulingPdf does not check either - would be better if both did! If I combine the endpoint as per Ed's comment below, then will need to do that anyway, so will all be resolved.